src/app/file-manager/services/login.service.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable } from '@angular/core';
+import { environment } from '@env/environment';
+import { Store } from '@ngrx/store';
+import { login} from '@app/core';
+import { ElectronService } from 'ngx-electron';
+import { SecurityService } from '../services/security.service';
+
+const googleConfig = {
+  clientId: environment.CLIENT_ID,
+  clientSecret: environment.CLIENT_SECRET,
+  authorizationUrl: environment.AUTHORIZATION_URL,
+  tokenUrl: environment.TOKEN_URL,
+  useBasicAuthorizationHeader: false,
+  redirectUri: environment.REDIRECT_URI
+};
+
+const googleOptions = {
+  scope: ['email',
+          'profile',
+          'https://www.googleapis.com/auth/compute',
+          'https://www.googleapis.com/auth/devstorage.full_control'].join(' '),
+  accessType: 'offline'
+};
+
+@Injectable()
+export class LoginService {
```

Comment:
We will need a mocked version of this service for testing. You can do this now or it can wait until the a test needs it.

Reply:
If it's ok for you, we could do this on the next sprint so we can integrate this important functionality on develop, and we could create a story on Jira to track it down.

Reply:
Sure, that would be fine.